src/app/shared/form/builder/ds-dynamic-form-ui/models/sponsor-autocomplete/ds-dynamic-sponsor-autocomplete.component.ts

```diff
@@ -70,7 +70,7 @@ export class DsDynamicSponsorAutocompleteComponent extends DsDynamicAutocomplete
     if (suggestion instanceof ExternalSourceEntry) {
       // suggestion from the openAIRE
       fundingProjectCode = this.getProjectCodeFromId(suggestion?.id);
-      fundingName = suggestion.metadata?.['project.funder.name']?.[0]?.value;
+      fundingName = suggestion.metadata?.['dc.title']?.[0]?.value;
```

Comment:
sure? `dc.title` seems to be something completely different to `project.funder.name`

Reply:
Yeah, that is that fix I did. It showed Funder name in the complex input field suggestion instead of project name.